crates/pallets/starknet/src/lib.rs

```diff
@@ -395,9 +391,16 @@ pub mod pallet {
             let fee_token_address = Self::fee_token_address();
             let state = &mut Self::create_state_reader()?;
             let call_info = transaction.execute(state, block, TxType::InvokeTx, None, fee_token_address);
+            let receipt;
             match call_info {
                 Ok(Some(mut v)) => {
-                    Self::emit_events(&mut v, &mut transaction).map_err(|_| Error::<T>::EmitEventError)?;
+                    let events = Self::emit_events(&mut v, &mut transaction).map_err(|_| Error::<T>::EmitEventError)?;
+                    receipt = TransactionReceiptWrapper {
+                        events: BoundedVec::try_from(events).unwrap(),
+                        transaction_hash: transaction.hash,
+                        tx_type: TxType::InvokeTx,
+                        actual_fee: U256::zero(), // TODO: switch to actual fee
```

Comment:
please create issue and link here

Reply:
https://github.com/keep-starknet-strange/madara/issues/251